pebblo/app/daemon.py

```diff
@@ -1,22 +1,47 @@
+from contextlib import redirect_stderr, redirect_stdout
 import uvicorn
 from fastapi import FastAPI
-from pebblo.app.routers.routers import router_instance
-
-from pebblo.topic_classifier.topic_classifier import TopicClassifier
-from pebblo.entity_classifier.entity_classifier import EntityClassifier
-
+from io import StringIO
+from tqdm import tqdm
```

Comment:
I suppose, all three libraries are either present by default in python 3.9+ or already present as dependency of some other package and no need to add them to requirements.txt/pyproject.toml.